airbyte-workers/src/main/java/io/airbyte/workers/general/DefaultReplicationWorker.java

```diff
@@ -305,18 +308,26 @@ private static Runnable getReplicationRunnable(final AirbyteSource source,
           }
           if (messageOptional.isPresent()) {
             if (messageOptional.get().getRecord() != null) {
-              try {
-                recordSchemaValidator.validateSchema(messageOptional.get().getRecord());
-              } catch (final RecordSchemaValidationException e) {
-                final ImmutablePair<String, Integer> exceptionWithCount = validationErrors.get(e.stream);
-                if (exceptionWithCount == null) {
-                  if (validationErrors.size() < 100) {
-                    validationErrors.put(e.stream, new ImmutablePair<>(e.getMessage(), 1));
+              // the stream this message corresponds to, including the stream namespace
+              final String messageStream = String.format("%s" + messageOptional.get().getRecord().getStream(),
+                  Objects.toString(messageOptional.get().getRecord().getNamespace(), ""));
```

Comment:
A couple comments here:
- I don't think the `String.format` is really necessary here; this could simply be `Objects.toString(messageOptional.get().getRecord().getNamespace(), "") + messageOptional.get().getRecord().getStream()`
- This logic is also repeated [here](https://github.com/airbytehq/airbyte/blob/417bdc1cb2b99bab9c8c4e61730256887e3042ca/airbyte-workers/src/main/java/io/airbyte/workers/WorkerUtils.java#L107-L109). Could you pull this out into a method which takes in a `@Nullable String namespace` and a `String streamName`, and returns the combined result? This can then be called from both `mapStreamNamesToSchemas()`, and here in DefaultReplicationWorker, to ensure that we are always combining these in the same way.